### ISSUE TITLE
Administrator Console

### ISSUE THREAD

**Issue by Timvrakas** [leinad2018/ProgrammingCompetition#3]
There should be a better way interact with the server from multiple admin locations.

**Comment by SecondThread** [comment]
Yeah, this will definitely be helpful if we have people actually asking questions through the request clarification tab, because we will probably need at least two people answering questions or offering public service announcements if people are continually getting questions wrong or if questions are poorly worded
